lib/reporter/cli.js

```diff
@@ -66,9 +66,9 @@ class Cli extends Base {
         }
       });
 
-      event.dispatcher.on(event.step.started, function (step) {
-        output.step(step, global.suiteDetails);
-      });
+      event.dispatcher.on(event.step.started, (step) => output.step(step));
+      event.dispatcher.on(event.step.passed, (step) => output.stepExecutionTime(step));
```

Comment:
Yes, output can get broken in --verbose and --debug mode (if something is printed during debug)
That may happen. And I don't have any solution to that (

Let's have a different format then.

In `--debug`: 

```
• I am on page "https://github.com/404" (9.716 sec)
```

in `--verbose`:

```
 • I am on page "https://github.com/404"
   Emitted | step.passed (I am on page "https://github.com/404")
   Finished in 9.716 sec
```